src/System.Diagnostics.Process/System.Diagnostics.Process.sln

```diff
@@ -24,8 +24,8 @@ Global
 		Windows_Release|Any CPU = Windows_Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{63634289-90D7-4947-8BF3-DBBE98D76C85}.Linux_Debug|Any CPU.ActiveCfg = Linux_Debug|Any CPU
-		{63634289-90D7-4947-8BF3-DBBE98D76C85}.Linux_Debug|Any CPU.Build.0 = Linux_Debug|Any CPU
+		{63634289-90D7-4947-8BF3-DBBE98D76C85}.Linux_Debug|Any CPU.ActiveCfg = Windows_Release|Any CPU
+		{63634289-90D7-4947-8BF3-DBBE98D76C85}.Linux_Debug|Any CPU.Build.0 = Windows_Release|Any CPU
```

Comment:
Yes this does seem to have been edited by accident. 

However I was thinking of making all the projects have default configuration set to windows, as I would expect more people to consume it that way. Let me know what you think and accordingly I can revert this file or keep as-is.